### PR TITLE
Fix/progress bar issue

### PR DESCRIPTION
**Pull request by mwazir**
PR to merge to main

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -10,11 +10,11 @@ import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
+  // <React.StrictMode>
```
Comment (benbourgon) [inline src/index.js:13]: death to Strict Mode

**Review by benbourgon — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (alextfriedman) [reply src/index.js:13]: 🔥

**Review by itsninam — APPROVED** [review]

**Review by alextfriedman — APPROVED** [review]
💥

File: src/index.js (hunk 1 above)
Comment (mwazir) [reply src/index.js:13]: 😂

**Pull request merged**